demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py

```diff
@@ -104,6 +104,20 @@ class StructureError(BaseStrictModel):
     error_type: Optional[str] = Field(None, alias="type")
     ctx: Optional[dict] = None
 
+    def __str__(self):
+        if self.error_type == "assertion_error":
+            self.error_message = f"The field {self.field_name} is not required, but should not be None if it exists"
```

Comment:
don't modify `self`. instead, use a local variable to store the error message and then print it in the end. (or print the raw `self.error_message` in the `else` case)

Reply:
Ok